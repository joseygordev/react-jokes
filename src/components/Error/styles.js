export default {
  root: {
    textAlign: 'center',
    lineHeight: 1.6,
    color: '#000'
  }
};