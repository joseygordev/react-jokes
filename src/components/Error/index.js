import React from 'react';

// Styles
import styles from './styles';

const Error = () => {
  return (
    <h3 style={styles.root}>Something went wrong :( <br /> please try again</h3>
  );
};

export default Error;
