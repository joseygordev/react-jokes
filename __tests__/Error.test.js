import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../src/components/Error';

describe('Error', () => {
  it('Should return the Error component', () => {
    const tree = renderer
      .create(<Error />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
