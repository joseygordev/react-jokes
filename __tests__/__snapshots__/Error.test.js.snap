// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Should return the Error component 1`] = `
<h3
  style={
    Object {
      "color": "#000",
      "lineHeight": 1.6,
      "textAlign": "center",
    }
  }
>
  Something went wrong :( 
  <br />
   please try again
</h3>
`;
